src/main/kotlin/blackjack/step2/domain/GameManager.kt

```diff
@@ -0,0 +1,17 @@
+package blackjack.step2.domain
+
+import blackjack.step2.domain.Cards.Companion.BLACKJACK_SCORE
+
+class GameManager(private val dealer: Dealer) {
+    fun dealFirstCards(players: Players): PlayerCards {
+        return dealer.dealFirstCards(players)
+    }
+
+    fun dealCardIfValid(playerCard: PlayerCard): PlayerCard {
+        if (playerCard.calculateScore() >= BLACKJACK_SCORE) {
+            println("${playerCard.playerName}의 점수가 21 이상입니다. 카드를 더 받을 수 없습니다.")
```

Comment:
```suggestion
            println("${playerCard.playerName}의 점수가 $BLACKJACK_SCORE 이상입니다. 카드를 더 받을 수 없습니다.")
```
이미 정의한 상수를 활용해보면 어떨까요?